vcd_cli/role.py

```diff
@@ -21,6 +22,9 @@ def role(ctx):
             
         vcd role list_rights
             Get list of rights associated with a given role.
+        
+        vcd role create role1 description 'Disk: View Properties' --org myOrg
+            Create a role with zero or more rights in the specified Org or current Org in use.
```

Comment:
I think its good to have an example where we can pass multiple rights. At present, i am not sure how to send more than one right looking at this example.

Reply:
fixed.